packages/cli/src/commands/serve.js

```diff
@@ -4,6 +4,7 @@ import fs from 'fs-extra'
 import terminalLink from 'terminal-link'
 
 import { recordTelemetryAttributes } from '@redwoodjs/cli-helpers'
+import { coerceRootPath } from '@redwoodjs/fastify-web/helpers'
```

Comment:
@Tobbe I sent you a message about this idea so that we could deduplicate it everywhere; let me know what you think